cves/CVE-2013-0867.yml

```diff
@@ -296,8 +338,11 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
-    note:
+    applies: true
+    note: |
+      Defense in Depth can serve as a good protection for this vulnerability as
+      it will protect the asset from unauthorized or improper modification to
```

Comment:
How exactly would defense in depth protect the asset from being modified?

Reply:
Defense in Depth will only accept known good inputs as a thorough input validation strategy. This vulnerability comes from not checking or validating input correctly, so taking valid inputs will block modifying the assets.